### DISCUSSION TITLE
Root CA for Parental Control Block Page

### DISCUSSION THREAD

**Discussion by WildByDesign** [AdguardTeam/AdGuardHome#6605] [Q&A]
Does AdGuard Team have a Root CA cert to install on iOS devices to handle the redirections used by Parental Control and Safe Browsing services?

I would like to achieve that similar to how OpenDNS Family Shield does it so that iOS device users land on the block page smoothly without certificate errors.

Thank you.

**Comment by ainar-g** [comment]
Not as far as I know.  Can you elaborate on that OpenDNS Family Shield certificate?  I wasn't able to find any info on that.

**Reply by WildByDesign** [reply]
Yes, absolutely. It’s the Cisco Umbrella Root CA for https block page redirections.

Link: https://support.opendns.com/hc/en-us/articles/227987007-Block-Page-Errors-Installing-the-Cisco-Umbrella-Root-CA

It works for kids devices when they hit up the block page but I assume it also works for when Cisco users setup custom block page messages as well.

It makes the whole block page redirect much smoother, especially iOS devices which complain the most.

**Reply by ainar-g** [reply]
Ah, I think I understand.  It's probably generating a certificate on-the-fly.  That's #578.

**Reply by WildByDesign** [reply]
Thank you, sir. I just added a comment to that issue right now to give it a bit of a boost.

With most of the web on https now, that old issue is definitely more relevant now. It’s difficult to ever get to the AdGuard block page on iOS due to certificate errors.

Although I can imagine generating certificates on-the-fly would not be an easy task, unfortunately.
